git-repo-env/README.md

```diff
@@ -3,6 +3,9 @@
 ## How to setup W261 git environment
 
 - Follow these steps to point your local repo to remote personal repo:
+git remote rename origin upstream
```

Comment:
Can you make sure that these render as a list. Right now they are all displayed in a single line.

Reply:
@kyleiwaniec , updated, please review again.